src/views/index.html

```diff
@@ -31,7 +31,7 @@ <h2 id="splash-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="pixel"
                             <div class="five unit column">
                                 <div id="splash-intro-text">
                                     <p>
-                                        In January 2018, Cambridge University’s annual hackathon will yet again bring together 300 Hackers, Programmers, Computer Scientists, Engineers, and more, from universities all over the world.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
+                                        In January 2018, Cambridge University’s annual hackathon will yet again bring together 300 hackers, programmers, designers, and more, from universities all over the world.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
```

Comment:
Not a blocker but I don't know if all these commas are necessary.

Reply:
It's the good ol' Oxford comma ;)